### ISSUE TITLE
Mention utf8mb4 encodig as db charset for emoji support, or just use it as default?

### ISSUE THREAD

**Issue by buddh4** [buddh4/humhub-docs#8]

**Comment by Felli** [comment]
I think that would be a alright idea, although I refer to this post for most of the information that I've gathered.
https://stackoverflow.com/questions/766809/whats-the-difference-between-utf8-general-ci-and-utf8-unicode-ci

**Comment by buddh4** [comment]
Yes I've also read that utf8mb4 is better and does not take more memory vor `VARCHAR` etc. So I think we should keept it as default.
